ChangeLog.md

```diff
@@ -2,6 +2,20 @@
 
 ## Unreleased changes
 
+## 0.2.0.0
+
+- [#12](https://github.com/parsonsmatt/annotated-exception/pull/12)
+    - Removed the `Eq` instance for `Annotation` as well as the `Eq` constraint
+      in `AnnC`. These instances were only used for testing, and prevented the
```

Comment:
> These instances were only used for testing,

I wish there were a way to expose something like this for testing without influencing the library interface, other than like CPP.